uint8 metode_pembayaran

uint8 rfid_flag
string rfid_tid

uint8 no_gardu_entrance
uint8 no_gerbang_entrance
uint8 no_gardu_exit
uint8 no_gerbang_exit
uint8 entrance_year
uint8 entrance_month
uint8 entrance_day
uint8 entrance_hour
uint8 entrance_minute
uint8 entrance_second
uint8 exit_year
uint8 exit_month
uint8 exit_day
uint8 exit_hour
uint8 exit_minute
uint8 exit_second

uint8 transaction_year
uint8 transaction_month
uint8 transaction_day
uint8 transaction_hour
uint8 transaction_minute
uint8 transaction_second

uint8 report_year
uint8 report_month
uint8 report_day

uint8 kode_ruas

uint8 no_shift
uint8 no_perioda
uint32 no_resi
uint32 no_kspt
uint32 no_plt

string hash

---

bool success
string response